poc/flp_generic.py

```diff
@@ -458,11 +455,8 @@ class ParallelSum(Gadget):
     """
 
     # Operational parameters
-    subcircuit = None
-    count = None
-
-    ARITY = None  # Set by constructor
-    DEGREE = None  # Set by constructor
```

Comment:
Reviewer note: this is an error because `ARITY` and `DEGREE` are declared to have type `int`. Similarly below.